pyquest/drawer.py

```diff
@@ -172,21 +294,23 @@ def get_circuit_columns(gates):
     num_qubits = get_num_qubits(gates)
 
     # {qubit index: column}
-    columns_of_last_target = {i:-1 for i in range(num_qubits)}
+    columns_of_last_target = {i: -1 for i in range(num_qubits)}
 
     # {qubit index: [columns]}
-    columns_occluded_by_connectors = {i:[] for i in range(num_qubits)}
+    columns_occluded_by_connectors = {i: [] for i in range(num_qubits)}
 
     for gate in gates:
 
         # all qubits controlled or targeted by gate (global operators return all)
         gate_qubits = get_operated_qubits(gate, num_qubits)
 
         # there must be room for all qubits between those explicitly targeted, for connectors
-        gate_range = (min(gate_qubits), max(gate_qubits)+1)
+        gate_range = (min(gate_qubits), max(gate_qubits) + 1)
 
         # find the leftmost column which fits the gate
-        gate_column = get_gate_column(gate_qubits, columns_of_last_target, columns_occluded_by_connectors)
```

Comment:
while here, you expand the function call from one lines to three! And further...